File: src/test/fixtures/kotlin/AnonymousCallables.kt
fun named(value: Int): Int = value

val moduleBound = { left: Int, right: Int -> left + right }

class Operations {
    val classBound = { value: Int -> value + 1 }

    fun namedMethod(value: Int): Int = value
}

fun outer(items: List<Int>): List<Int> {
    val localBound = { value: Int -> value * 2 }
    return items.map { it + 1 }
}

File: src/test/fixtures/fsharp/AnonymousCallables.fs
let named value = value

let rec mutualFirst value = value
and mutualSecond left right = left + right

let moduleBound = fun left right -> left + right

type Operations() =
    static member ClassBound = fun value -> value + 1

    member _.NamedMethod value = value

let outer items =
    let localBound = fun value -> value * 2
    items |> List.map (fun item -> item + 1)
